wordpress-mobile/WordPress-iOS
# Issue/9839 Implement Rich Notification Scaffolding

Addresses #9839, with the following changes:

1. Creates the notification service extension.
1. Modifies build settings to match the provisioning implemented for both Apple Developer Portal instances (i.e., Retail & Enterprise) of _Certificates, Identifiers & Profiles_.
1. Organizes the internal contents of the extension.
1. Creates placeholder files for Base & English localizations of `Localizable.strings`

To test:

1. Checkout branch. Validate app builds & existing tests pass.
1. Review each commit for apparent correctness.
1. The Xcode template appends `[modified]` to the notification title. Manually exercising the service extension with a WPCOM sandbox appending `mutable-content : 1` to the payload should make that visible.

When run, a payload like the following:
```
{"aps":{"alert":"Test (6)","badge":1,"content-available":1,"sound":"default", "mutable-content":1 }}
```

should evince this the following on-device:
![9874](https://user-images.githubusercontent.com/221062/43427095-04235f8e-940d-11e8-8d1f-7934cd06bd50.png)
